lib/http_client/http_client.ex

```diff
@@ -22,6 +23,8 @@ defmodule ElixirTools.HttpClient do
   @type action :: :get | :create | :update
   @type adapter :: module
 
+  @default_response_timeout 10_000
```

Comment:
I would say we most likely want a bigger default value, for a lot of external services(Paypal, CECA etc.) it's not rare to see bigger response times